### PR TITLE
Add hyperlinks to port scan URLs

### PR DESCRIPTION
**Pull request by vincentchu12**
## Contributor License Agreement

I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

## Task list

- [x] Agree to Contributor License Agreement
- [ ] Check to see if it looks okay locally. `hugo server --minify`

### CODE REVIEW

File: content/post/initial-port-scan.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ PORT | STATE | SERVICE
 
 ## GEO Vision - GV-SNVR0811
```
Comment (vincentchu12) [inline content/post/initial-port-scan.md:60]: https://ctf.iotvillage.org/challenges#Geovision%20GV-SNVR0811-3

File: content/post/initial-port-scan.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ PORT | STATE | SERVICE
 
 ## NUUO Network Video Recorder
```
Comment (vincentchu12) [inline content/post/initial-port-scan.md:39]: https://ctf.iotvillage.org/challenges#Nuuo%20NVRMini%202-64

File: content/post/initial-port-scan.md [hunk 3]
```diff
@@ -84,7 +84,7 @@ PORT | STATE | SERVICE
 
 ## DCS-930LB1
```
Comment (vincentchu12) [inline content/post/initial-port-scan.md:85]: https://ctf.iotvillage.org/challenges#D-Link%20DCS-930LB1-52

File: content/post/initial-port-scan.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ nmap 192.168.10.0/24
 
 ## Control4 Open Source Software Notice
```
Comment (vincentchu12) [inline content/post/initial-port-scan.md:19]: https://ctf.iotvillage.org/challenges#Control4%20C4-HC250-BL-67

**Review by mariahnorell — APPROVED** [review]

**Pull request merged**